packages/adblocker/src/engine/engine.ts

```diff
@@ -525,6 +533,8 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
     this.importants = new NetworkFilterBucket({ config: this.config });
     // $redirect
     this.redirects = new NetworkFilterBucket({ config: this.config });
+    // $removeparam
+    this.substitudes = new NetworkFilterBucket({ config: this.config });
```

Comment:
If the bucket is specifically about removeparam, why give it another name?
```suggestion
    // $removeparam
    this.removeparams = new NetworkFilterBucket({ config: this.config });
```